src/SignalRServiceExtension/Bindings/ServiceEndpointExtensions.cs

```diff
@@ -0,0 +1,28 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Microsoft.Azure.SignalR;
+
+namespace Microsoft.Azure.WebJobs.Extensions.SignalRService
+{
+    internal static class ServiceEndpointExtensions
+    {
+        private const string FakeAccessKey = "ABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789";
+        public static LiteServiceEndpoint ToLiteServiceEndpoint(this ServiceEndpoint e)
+        {
+            return new LiteServiceEndpoint
+            {
+                EndpointType = e.EndpointType,
+                Name = e.Name,
+                Endpoint = e.Endpoint,
+                Online = e.Online
+            };
+        }
+
+        public static ServiceEndpoint ToEqualServiceEndpoint(this LiteServiceEndpoint e)
+        {
+            var connectionString = $"Endpoint={e.Endpoint};AccessKey={FakeAccessKey};Version=1.0;";
```

Comment:
>FakeAccessKey [](start = 70, length = 13)

if cx use `ServiceEndpoint`, then they will have endpoint with fake access key. they will be confused of all the access key are the same (fake key), while they are different what they get from portal